services/webnn/ort/ort_model_builder.h

```diff
@@ -59,6 +59,16 @@ class OrtModelBuilder final {
                       base::span<const uint8_t> data,
                       ONNXTensorElementDataType data_type);
 
+  void AddInitializerAsRawData(std::string_view name,
```

Comment:
Will it be better if you add some comments for why you add these two methods and how to choose one from these three methods to add initialier?